src/tests/Microsoft.PowerFx.Interpreter.Tests/SetFunctionTests.cs

```diff
@@ -1,6 +1,8 @@
 ﻿// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+using System;
+using System.Threading;
```

Comment:
Are these being used?

Reply:
Not system. But Threading is used for CancellationToken